### PR TITLE
Log build logs for searchability

### PR DESCRIPTION
**Pull request by mandrean**
This PR introduces some changes in order to print the actual Jenkins build logs (from stages section of the build data) to stdout.

It would enable one to search the actual Jenkins build logs in e.g. Google Cloud Logging/Stackdriver, Kibana etc.

cc @jeffersongirao

### CODE REVIEW

**Comment by alde** [comment]
I think this will output a lot of information (as it will crawl the current state of the build every 5 seconds until it's finished)

**Comment by mandrean** [comment]
Thanks @alde! Good catch.

I moved the logic so that it executes after all crawling has finished now

**Review by jeffersongirao — APPROVED** [review]

File: jenkins/crawler.go [hunk 1]
```diff
@@ -56,30 +60,75 @@ func (c *Crawler) CrawlJenkins(buildURI string, buildID string) {
...
+			c.log.WithField("build_id", buildID).Info("database updated")
 
 			if jdata.Status == "" || jdata.Status == "IN_PROGRESS" {
 				go func() {
-					logrus.Debug("sleeping for 5 seconds before requerying")
+					c.log.Debug("sleeping for 5 seconds before requerying")
 					time.Sleep(5 * time.Second)
 					c.processChannel <- buildID
 				}()
 			} else {
```
Comment (mandrean) [inline jenkins/crawler.go]: I'm thinking of removing the `else` block here. WDYT? @alde @jeffersongirao 

image

Edit: Last line should be `return`.

**Comment by alde** [comment]
I wonder if this should also be added to the config, so you can decide if you want it or not?

**Pull request merged**